src/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To-Do List</title>
+  <link rel="stylesheet" href="./style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css">
+</head>
+<body>
+  <header>
+      <h1>Today's To Do</h1>
+      <i class="fa-solid fa-rotate"></i>
+  </header>
+  <ul id="task-list"></ul>
+  <button id="clear-btn">Clear Completed Tasks</button>
+  <script src="/dist/main.js"></script>
+</body>
```

Comment:
- As requested by the previous reviewer [here](https://github.com/Christelle-12/To-do-list/pull/1#discussion_r1160560923) kindly style your project as requested in the project requirements [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) :recycle: